File: BasicSortingAlgorithms/SelectionSort/sort.cpp
#include <vector>
using namespace std;

class Solution {
  public:
    // Function to perform selection sort on the given array.
    void selectionSort(vector<int> &arr) {
        // code here
        int n = arr.size();
        for(int i=0;i<n-1;i++){
            int idx = i;
            for(int j=i+1;j<n;j++){
                if(arr[idx] > arr[j]){
                    idx = j;
                }
            }
            swap(arr[idx],arr[i]);
        }
    }
};

File: BasicSortingAlgorithms/SelectionSort/sort.py
class Solution: 
    def selectionSort(self, arr):
        #code here
        n = len(arr)
        for i in range(n-1):
            idx = i
            for j in range(i+1,n) :
                if arr[idx] > arr[j]: 
                    idx = j
            arr[idx],arr[i] = arr[i],arr[idx]